test/unit/alignment/aligned_sequence_test_template.hpp

```diff
@@ -27,7 +27,7 @@ class aligned_sequence_ : public ::testing::Test
 
 seqan3::dna4_vector const seq = "ACTA"_dna4;
 
-TYPED_TEST_CASE_P(aligned_sequence_);
+TYPED_TEST_SUITE_P(aligned_sequence_);
 
```

Comment:
When exactly do you need the ", " because sometimes you don't write one. That stands for a certain parameter?

Reply:
It's a bug that only affects `INSTANTIATE_TYPED_TEST_SUITE_P`  https://github.com/google/googletest/issues/2271